beta-src/src/enums/ArrowType.ts

```diff
@@ -1,13 +1,8 @@
 enum ArrowType {
-  CONVOY = "convoy",
-  FOREIGN_IMPLIED = "foreignImplied",
-  IMPLIED = "implied",
-  MOVE_ORDER_SELECTED = "moveOrderSelected",
-  MOVE = "move",
-  MOVE_FAILED = "moveFailed",
-  MOVE_SUPPORT = "moveSupport",
-  HOLD_SUPPORT = "holdSupport",
-  RETREAT = "retreat",
+  CONVOY,
+  MOVE,
+  SUPPORT,
+  HOLD,
```

Comment:
Will we be adding back the arrow types that were removed?

Reply:
no